prow/spyglass/viewers/junit/junit_viewer.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package junit provides a junit viewer for Spyglass
+package junit
+
+import (
+	"bytes"
+	"fmt"
+
```

Comment:
nit: follow the grouping convention used elsewhere in the prow codebase (stdlib, third-party, test-infra)
```go
import (
	"bytes"
	"fmt"
	"html/template"

	junit "github.com/joshdk/go-junit"
	"github.com/sirupsen/logrus"

	"k8s.io/test-infra/prow/spyglass/viewers"
)
```